### PR TITLE
pr split 11 - add 10 files

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/tzip16-originate-contract-having-metadata-on-IPFS-and-fetch-those.spec.ts [hunk 1]
```diff
@@ -16,13 +16,13 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
     let contractAddress: string;
 
-    describe(`Originating contracts having metadata stored at HTTPS URL using: ${rpc}`, () => {
+    describe(`Test contract origination having metadata stored at HTTPS URL through contract api using: ${rpc}`, () => {
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-originate-contract-having-metadata-on-IPFS-and-fetch-those.spec.ts:19]: Should be IPFS not HTTP

File: integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts [hunk 2]
```diff
@@ -13,13 +13,13 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
     const test = require('jest-retries');
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts:14]: should be removed ` const test = require('jest-retries');`?

File: integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts [hunk 3]
```diff
@@ -119,7 +119,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
             done();
         });
 
-        test('Fetch the metadata which contains emoji of the contract', 2, async (done: () => void) => {
+        test('Verify that the metadata for the contract which contains emoji of the contract can be fetched', 2, async (done: () => void) => {
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts:122]: 'Verify that the metadata for the contract which contains emoji ~of the contract~ can be fetched

File: integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts [hunk 4]
```diff
@@ -11,14 +11,14 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
     const test = require('jest-retries');
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts:12]: should be removed?

File: integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts [hunk 5]
```diff
@@ -11,14 +11,14 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
...
 
-    describe(`Originating contracts having metadata stored at HTTPS URL using: ${rpc}`, () => {
+    describe(`Test contract origination having having a sha256 hash in URI through contract api using: ${rpc}`, () => {
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts:14]: having having

File: integration-tests/tzip16-originate-wallet-having-metadata-on-IPFS-and-fetch-those.spec.ts [hunk 6]
```diff
@@ -17,13 +17,13 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
     let contractAddress: string;
 
-    describe(`Originating contracts made with wallet api having metadata stored at HTTPS URL using: ${rpc}`, () => {
+    describe(`Test contract origination having metadata stored at HTTPS URL through wallet api using: ${rpc}`, () => {
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-originate-wallet-having-metadata-on-IPFS-and-fetch-those.spec.ts:20]: Should be IPFS not HTTPS

File: integration-tests/tzip16-originate-wallets-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts [hunk 7]
```diff
@@ -114,7 +114,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
             done();
         });
 
-        it('Fetch the metadata which contains emoji of the wallet api contract', async (done) => {
+        it('Verify that the metadata for the contract which contains emoji of the contract can be fetched', async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-originate-wallets-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts:117]: Verify that the metadata for the contract which contains emoji ~of the contract~ can be fetched

File: integration-tests/tzip16-originate-contract-having-metadata-on-IPFS-and-fetch-those.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/tzip16-originate-contract-having-metadata-on-IPFS-and-fetch-those.spec.ts:19]: Changed to IPFS

File: integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts (hunk 3 above)
Comment (michaelkernaghan) [reply integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts:122]: Changed to:
'Verify that the metadata for the contract which contains emoji can be fetched.'

File: integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts (hunk 4 above)
Comment (michaelkernaghan) [reply integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts:12]: Removed that line and the retry calls in each test.

File: integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts (hunk 5 above)
Comment (michaelkernaghan) [reply integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts:14]: Too much having!  I halved it :)

File: integration-tests/tzip16-originate-wallet-having-metadata-on-IPFS-and-fetch-those.spec.ts (hunk 6 above)
Comment (michaelkernaghan) [reply integration-tests/tzip16-originate-wallet-having-metadata-on-IPFS-and-fetch-those.spec.ts:20]: Changed to IPFS.
Also removed jest retries

File: integration-tests/tzip16-originate-wallets-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts (hunk 7 above)
Comment (michaelkernaghan) [reply integration-tests/tzip16-originate-wallets-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts:117]: Changed to 
'Verify that the metadata for the contract which contains emoji can be fetched'

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**